src/Property.java

```diff
@@ -0,0 +1,43 @@
+/**
+ *
+ */
+public class Property {
+    private String name;
+    private String color;
+    private int price;
+    private int mortgage;
+    private boolean isowned;
```

Comment:
variable names need to be camel cased.  This should be `isOwned` instead of `isowned`